### DISCUSSION TITLE
The model: `gpt-4` does not exist

### DISCUSSION THREAD

**Discussion by BathmanTv** [microsoft/pyright#5037] [General]
Hello team i'm losing some input regarding the jobs,
Even when i tried to use python -m multigpt --speak --gpt3only i get all error (i give you the msg error i have in PJ)
[LogError.txt](https://github.com/microsoft/pyright/files/11361139/LogError.txt)

**Comment by erictraut** [comment]
This problem does not appear to have anything to do with pyright or static type checking. If you have questions about a particular Python library, please find the associated project and ask your question there.
